### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by 0x6675636b796f75676974687562**

### CODE REVIEW

File: save-deploy/README.md [hunk 1]
```diff
@@ -183,19 +183,128 @@ Usually, not the whole stack is required for development. Application logic is p
...
+```
+
+and provide the path to the JAR archive which contains `save-agent.kexe` via the
+`saveAgentDistroFilepath` _Gradle_ property, by setting the above property
+either under project-specific `gradle.properties`, or, globally, under
+`%USERPROFILE%\.gradle\gradle.properties`, e.g.:
+
+```properties
+# gradle.properties
+saveAgentDistroFilepath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cloud\\save-agent\\build\\libs\\save-agent-0.3.0-alpha.0.48+1c1fd41-distribution.jar
```
Comment (nulls) [inline save-deploy/README.md:204]: maybe refactor `gradle build` to support system environment to avoid changing `gradle.properties` ?

Comment (nulls) [inline save-deploy/README.md:204]: also, why we need a link to `WSL`?
I guess that it's expected that we built on WSL the same project (I mean the location on a disk) as on Windows

File: save-deploy/README.md [hunk 2]
```diff
@@ -183,19 +183,128 @@ Usually, not the whole stack is required for development. Application logic is p
...
+1. Reload the project from disk (_Project_ tool window in IDEA).
+1. Reload the project model (_Gradle_ tool window in IDEA).
+1. Re-start the _back-end_ application.
+
+Then verify that the agent is indeed available for download from the _back-end_.
+Incorrect output example (the binary is not found):
+
+```console
+$ curl -vvv -X POST http://localhost:5800/internal/files/download-save-agent --output save-agent.kexe
+*   Trying 127.0.0.1:5800...
```
Comment (akuleshov7) [inline save-deploy/README.md:236]: may be remove this information? I think people will see it when they will run it

File: save-deploy/README.md [hunk 3]
```diff
@@ -183,19 +183,128 @@ Usually, not the whole stack is required for development. Application logic is p
...
 
 If you need to test changes in `save-cli` you can also compile `SNAPSHOT` version of `save-cli` on WSL <br/>
 and set `saveCliPath` and `saveCliVersion` in `%USERPROFILE%\.gradle\gradle.properties` <br/>
 For example:
 
-    saveCliPath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cli\\save-cli\\build\\bin\\linuxX64\\releaseExecutable
-    saveCliVersion=0.4.0-alpha.0.42+78a24a8
+```properties
+# gradle.properties
+saveCliPath=file:\\\\\\\\wsl$\\Ubuntu\\home\\username\\projects\\save-cli\\save-cli\\build\\bin\\linuxX64\\releaseExecutable
```
Comment (nulls) [inline save-deploy/README.md:305]: the same question about `WSL`

**Review by akuleshov7 — APPROVED** [review]

**Review by nulls — APPROVED** [review]
approved with some questions

File: save-deploy/README.md (hunk 1 above)
Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md:216]: Added a note on WSL with ee64d9814e1001d451bad76bfb5d0e6f194aad0b.

File: save-deploy/README.md (hunk 3 above)
Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md:317]: Added a note on WSL with ee64d9814e1001d451bad76bfb5d0e6f194aad0b.

File: save-deploy/README.md (hunk 2 above)
Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md]: Addressed with cd49c63011f418cd1724a05e5a7be13c852c7ab8

File: save-deploy/README.md (hunk 1 above)
Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md:216]: This is already possible: you can set Gradle properties either globally, or on a per-project basis.

So, if you want your local Git repository clean, you can just putt al the necessary data under `~/.gradle/gradle.properties`.

**Review by Cheshiriks — APPROVED** [review]

File: save-deploy/README.md (hunk 1 above)
Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md:216]: Additionally, you may set the property via the command line, e.g.:

```console
gradlew.bat gradlew.bat :save-backend:downloadSaveAgentDistro -PsaveAgentDistroFilepath=file:////wsl$/Debian/home/username/projects/save-cloud/save-agent/build/libs/save-agent-0.4.0-SNAPSHOT-distribution.jar
```

Comment (0x6675636b796f75676974687562) [reply save-deploy/README.md:216]: Further addressed with b5c13d99c8a62ffb05728b53a0522285c5e6e55e

**Pull request merged**